src/components/layout/error-reporting-notice/error-reporting-notice.tsx

```diff
@@ -1,66 +1,73 @@
-import { useRef, useState } from 'react';
-import { ERROR_REPORTING_CONSENT_KEY_NAME, images, isErrorReportingAllowed } from '../../../utils';
+import { useState } from 'react';
+import { images } from '../../../utils';
 import Button from '../../button';
-import { triggerOnEnter } from '../../modal';
 import styles from './error-reporting-notice.module.scss';
 import ExternalLink from '../../external-link';
+import { links } from '../../../utils/links';
+import { ALLOW_ANALYTICS, ALLOW_ERROR_REPORTING, initAnalytics } from '../../../utils/analytics';
+import PrivacySettingsModal from '../privacy-settings-modal';
+import { CrossIcon } from '../../icons';
 
 interface WelcomeModalContentProps {
-  onClose: () => void;
+  onShowNotice: (showNotice: boolean) => void;
 }
 
 const ErrorReportingNotice = (props: WelcomeModalContentProps) => {
-  const { onClose } = props;
-  const defaultReportingValue = localStorage.getItem(ERROR_REPORTING_CONSENT_KEY_NAME);
-  const [errorReportingEnabled, setErrorReportingEnabled] = useState(
-    defaultReportingValue === null || isErrorReportingAllowed(defaultReportingValue)
-  );
-  const onErrorReportingNoticeConfirm = () => {
-    localStorage.setItem(ERROR_REPORTING_CONSENT_KEY_NAME, errorReportingEnabled.toString());
-    onClose();
+  const { onShowNotice } = props;
+
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const handleSubmit = (allowAnalytics: boolean, allowReporting: boolean) => {
+    localStorage.setItem(ALLOW_ERROR_REPORTING, allowReporting.toString());
+    localStorage.setItem(ALLOW_ANALYTICS, allowAnalytics.toString());
+
+    if (allowAnalytics) {
+      initAnalytics();
+      (window as any).clarity?.('consent');
+    }
+
+    if (allowReporting) {
+      // initSentry(); // TBD later
```

Comment:
We should continue to report errors to sentry. Now we won't be doing it because in `src/index.tsx` we are still using the old  local storage key ("reportErrors")

Reply:
Oh, yeah. My bad.
It's fixed now. Took the initialisation logic from the Market for both analytics and sentry